lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -163,40 +163,43 @@ describe('ReportUIFeatures', () => {
     });
   });
 
+  // This is done all in CSS, but tested here.
```

Comment:
since this is self contained, cut and paste over to `performance-category-renderer.js`. Slightly better there?